src/components/Files/DischargeSummarySubTab.tsx

```diff
@@ -46,22 +46,20 @@ import {
 } from "@/common/constants";
 
 import routes from "@/Utils/request/api";
+import mutate from "@/Utils/request/mutate";
 import query from "@/Utils/request/query";
+import { HTTPError } from "@/Utils/request/types";
 import { formatName } from "@/Utils/utils";
-import ReportBuilderSheet from "@/pages/Encounters/ReportBuilder/ReportBuilderSheet";
-import { Encounter } from "@/types/emr/encounter";
 
 interface DischargeTabProps {
   type: "encounter" | "patient";
-  facilityId: string;
-  encounter: Encounter;
+  encounterId: string;
   canEdit: boolean | undefined;
 }
 
 export const DischargeTab = ({
   type,
-  facilityId,
-  encounter,
+  encounterId,
```

Comment:
let's keep facilityId and encounter and use them. Will make it easier to switch back in the future.